AlphaWallet/Localization/zh-Hans.lproj/Localizable.strings

```diff
@@ -408,3 +408,5 @@ wallets.backupHdWallet.introduction.title = "Back up your Wallet with Seed Phras
 "wallet.security.level.red" = "BACK UP NOW!";
 "wallet.security.level.orange" = "NOT LOCKED";
 "wallet.security.level.green" = "100% SECURE";
+
+"settings.general.title" = "settings.general.title";
```

Comment:
@James-Sangalli we'll need to duplicate this in the other 4 (?) `Localisable.strings` files:

```
"settings.general.title" = "General";
```

(and also not `"settings.general.title" = "settings.general.title";`)